apt/private/deb_translate_lock.bzl

```diff
@@ -23,20 +23,6 @@ _DEB_IMPORT_TMPL = '''\
     )
 '''
 
-_PACKAGE_TEMPLATE = """\
```

Comment:
Let's keep this as private, your change makes it reuse the public template, which will break usages such as; 

https://github.com/GoogleContainerTools/distroless/blob/main/private/repos/deb/package.BUILD.tmpl

Let's keep the PR focused, it says "add more convience targets", so it should be doing just that.